lib/matplotlib/tests/test_axes.py

```diff
@@ -3794,6 +3794,22 @@ def test_horiz_violinplot_custompoints_200():
                   showextrema=False, showmedians=False, points=200)
 
 
+@image_comparison(['violinplot_sides.png'])
```

Comment:
```suggestion
@image_comparison(['violinplot_sides.png'], remove_text=True, style="mpl20")
```

Small thing, but in adding new image tests we prefer using the newer style (but currently default to old style because of the volume of tests that predate the change)

We can also remove text here.

Note this will change the image as well. (Also I did not check line length in the suggestion so might need to be split into two lines)

Reference for our preferences:
https://matplotlib.org/devdocs/devel/testing.html#writing-an-image-comparison-test